From 6514729674c1054580cd85feca527740f5476813 Mon Sep 17 00:00:00 2001
From: David Hildenbrand <david@redhat.com>
Date: Tue, 17 Oct 2017 19:15:47 +0200
Subject: [PATCH 42/69] tools/kvm_stat: remove unnecessary header redraws

RH-Author: David Hildenbrand <david@redhat.com>
Message-id: <20171017191605.2378-22-david@redhat.com>
Patchwork-id: 77327
O-Subject: [RHEL-7.5 qemu-kvm-rhev PATCH 21/39] tools/kvm_stat: remove unnecessary header redraws
Bugzilla: 1497137
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Cornelia Huck <cohuck@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Thomas Huth <thuth@redhat.com>

Upstream-status: linux.git 2da9d4aaa7348fc13374d7398c9c7027b0a9e2cb

commit 2da9d4aaa7348fc13374d7398c9c7027b0a9e2cb
Author: Stefan Raspl <raspl@linux.vnet.ibm.com>
Date:   Wed Jun 7 21:08:28 2017 +0200

    tools/kvm_stat: remove unnecessary header redraws

    Certain interactive commands will not modify any information displayed in
    the header, hence we can skip them.

    Signed-off-by: Stefan Raspl <raspl@linux.vnet.ibm.com>
    Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>

Signed-off-by: David Hildenbrand <david@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 scripts/kvm/kvm_stat | 2 --
 1 file changed, 2 deletions(-)

diff --git a/scripts/kvm/kvm_stat b/scripts/kvm/kvm_stat
index 6e29e5b..d2526b6 100755
--- a/scripts/kvm/kvm_stat
+++ b/scripts/kvm/kvm_stat
@@ -1140,7 +1140,6 @@ class Tui(object):
             try:
                 char = self.screen.getkey()
                 if char == 'x':
-                    self.refresh_header()
                     self.update_drilldown()
                 if char == 'q':
                     break
@@ -1158,7 +1157,6 @@ class Tui(object):
                     self.show_vm_selection_by_pid()
                     sleeptime = DELAY_INITIAL
                 if char == 'r':
-                    self.refresh_header()
                     self.stats.reset()
             except KeyboardInterrupt:
                 break
-- 
1.8.3.1

